app/messages/index.js

```diff
@@ -0,0 +1,32 @@
+import util from 'util'
+import { messageConfig } from '../config/index.js'
+import { MessageReceiver } from 'ffc-messaging'
```

Comment:
Would recommend moving third party libraries to the top of the file like you have with `uitl`

Reply:
Done.